python/lsst/summit/utils/utils.py

```diff
@@ -454,7 +454,7 @@ def getSite():
     Returns
     -------
     location : `str`
-        One of ['tucson', 'summit', 'base', 'lsst-dev']
+        One of ['tucson', 'summit', 'base', 'rsp', 'lsst-dev']
```

Comment:
I might have called this `staff-rsp`; all of the deployments at the TTS, Summit, Base, and IDF are RSPs, which is a generic term.  Also note that this will move to USDF before August 15.

Reply:
Good point about changing to `staff-rsp`. RE: USDF migration, as discussed with you last week, I aim to remove all of this site-specific stuff _long_ before then (it's only TTS that actually uses this at present).

I'll fixup and self merge the update.

Reply:
Fixed on DM-34850.